src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java

```diff
@@ -180,19 +182,19 @@ void persistBatch(Batch batch) throws Exception {
                 aggregator.aggregate(cygnusEvent);
             } // for
             
-            // persist the fieldValues
+            // persist the aggregation
             persistAggregation(aggregator);
             batch.setPersisted(destination);
         } // for
     } // persistBatch
     
     /**
-     * Class for aggregating fieldValues.
+     * Class for aggregating.
      */
     private abstract class MySQLAggregator {
         
-        // string containing the data fieldValues
-        protected String aggregation;
+        // object containing the aggregted data
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/f43ad5018cb279a5da36a91bdd130afdd99590cc